Proposal-Process.md

```diff
@@ -0,0 +1,14 @@
+For a proposal to create or change a process foundation-wide, a PR should be
```

Comment:
Adding a title: That's the other thing I struggled with...what to call this thing. Does `Policy Change Proposal Process` imply that this only applies to "policies" (whatever those are) and not "guidelines" or "charters" or whatever?

I'm probably over-thinking this and will go with your suggestion, @jasnell.